index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>survey form</title>
+  </head>
+  <body>
+    <form id="survey-form">
+      <label id="name-label">Name:</label>
+      <input type="text" placeholder="Enter your name" required class="text" />
+
+      <label id="email-label">Email:</label>
+      <input
+        type="email"
+        placeholder="Enter your email address"
+        required
+        class="email"
+      />
+
+      <label id="number-label">Number:</label>
+      <input
+        type="number"
+        placeholder="Enter a number"
+        min="0"
+        max="100"
+        class="number"
+      />
+
+      <label id="dropdown-label">Select an option:</label>
+      <select>
+        <option value="option1">Option 1</option>
+        <option value="option2">Option 2</option>
+      </select>
```

Comment:
I think you should use the **for** attribute and **id**. This lets users click on the label to focus or activate the input field. Add unique **id** attributes to the input elements and reference them in the **for** attributes of the labels.. 😄